bin/wasm-node/javascript/src/index-browser.ts

```diff
@@ -267,13 +267,17 @@ export function start(options?: ClientOptions): Client {
             console.error("Local offer doesn't contain UDP data channel. WebRTC connections will likely fail. Please report this issue.");
           }
           // According to the libp2p WebRTC spec, the ufrag and pwd are the same
-          // randomly-generated string. We modify the local description to ensure that.
-          const pwd = sdpOffer.match(/^a=ice-pwd:(.+)$/m);
-          if (pwd != null) {
-            sdpOffer = sdpOffer.replace(/^a=ice-ufrag.*$/m, 'a=ice-ufrag:' + pwd[1]);
-          } else {
+          // randomly-generated string on both sides, and must be prefixed with
+          // `libp2p-webrtc-v1:`. We modify the local description to ensure that.
+          // While we could randomly generate a new string, we just grab the one that the
+          // browser has generated, in order to make sure that it respects the constraints
+          // of the ICE protocol.
+          const ufragPwd = "libp2p-webrtc-v1:" + sdpOffer.match(/^a=ice-pwd:(.+)$/m)?.at(1);
+          if (ufragPwd === undefined) {
```

Comment:
Note for myself: since we do `"libp2p-webrtc-v1:" + ...`, this can never actually be undefined